### PR TITLE
DEP PHP Support in CMS5

### PR DESCRIPTION
**Pull request by sabina-talipova**
## Parent issue
- https://github.com/silverstripe/silverstripe-framework/issues/10402

### CODE REVIEW

**Review by emteknetnz — CHANGES_REQUESTED** [review]
Unit test failure

**Comment by sabina-talipova** [comment]
Tests passed

File: code/SQLite3Query.php [hunk 1]
```diff
@@ -72,7 +72,7 @@ public function numRecords()
         return $c;
     }
 
-    public function nextRecord()
+    public function getIterator()
```
Comment (emteknetnz) [inline code/SQLite3Query.php:75]: https://github.com/silverstripe/silverstripe-framework/blob/5/src/ORM/Connect/MySQLQuery.php#L48Could you add the `#[\ReturnTypeWillChange]` above this method e.g.

**Review by emteknetnz — CHANGES_REQUESTED** [review]

File: code/SQLite3Query.php (hunk 1 above)
Comment (sabina-talipova) [reply code/SQLite3Query.php:76]: DONE

**Review by emteknetnz — APPROVED** [review]

**Pull request merged**